<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录页面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="IP_label">
       <property name="text">
        <string>服务器IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="user_label">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="database_label">
       <property name="text">
        <string>数据库名</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="user_text">
       <property name="text">
        <string>root</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="password_text"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="database_text"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="IP_text">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="login_button">
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
